// react-email/dist/utils/contants.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.PACKAGE_EMAILS_PATH = exports.CLIENT_EMAILS_PATH = exports.NODE_MODULES_PREVIEW_PATH = exports.NODE_MODULES_PACKAGE_PATH = exports.EVENT_FILE_DELETED = exports.REACT_EMAIL_ROOT = exports.CLIENT_PACKAGE_JSON = exports.HOME_DIR = exports.CURRENT_PATH = exports.PACKAGE_NAME = exports.DEFAULT_EMAILS_DIRECTORY = void 0;
const os_1 = __importDefault(require("os"));
const path_1 = __importDefault(require("path"));
// Package variables
exports.DEFAULT_EMAILS_DIRECTORY = 'emails';
exports.PACKAGE_NAME = 'create-react-email-test';
// Default paths
exports.CURRENT_PATH = process.cwd();
exports.HOME_DIR = os_1.default.homedir();
// Client paths
exports.CLIENT_PACKAGE_JSON = `${exports.CURRENT_PATH}/package.json`;
// React email paths
exports.REACT_EMAIL_ROOT = path_1.default.join(exports.HOME_DIR, '.react-email');
// Events
exports.EVENT_FILE_DELETED = 'unlink';
exports.NODE_MODULES_PACKAGE_PATH = path_1.default.join(exports.CURRENT_PATH, 'node_modules', exports.PACKAGE_NAME);
exports.NODE_MODULES_PREVIEW_PATH = path_1.default.join(exports.NODE_MODULES_PACKAGE_PATH, 'preview');
exports.CLIENT_EMAILS_PATH = path_1.default.join(exports.CURRENT_PATH, exports.DEFAULT_EMAILS_DIRECTORY);
exports.PACKAGE_EMAILS_PATH = path_1.default.join(exports.REACT_EMAIL_ROOT, exports.DEFAULT_EMAILS_DIRECTORY);
